apps/condo/domains/acquiring/utils/serverSchema/feeDistribution.js

```diff
@@ -8,17 +8,58 @@
     γ - commission from initial amount of payment, 0.004 ( 0.4 %)
 */
 
-/*  Fee settings
+/*
+    Differences between service fee and commission
+    Commission:
+        1. Is calculated based on the billingReceipt amount. For $1000 with commission 1.2% - commission will be $12
+        2. Can have minimum and maximum amount
+    Service fee:
+        1. Is calculated based on formula ↑. Because, acquiring get commission based on total payment (billingReceipt + serviceFee)
+        For $1000 with serviceFee 0.4% and acquiring fee 0.8% - additional amount will be (0.008 + 0.004) ⋅ $1000 / (1 - 0.008) = $12.10
```

Comment:
What is `(1 - 0.008)`?

```suggestion
        For $1000 with serviceFee 0.4% and acquiring fee 0.8% - additional amount will be (0.004 + 0.008) ⋅ $1000 / (1 - 0.008) = $12.10
```

Reply:
The formula is on the top of file